src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PRMigratorBar.xaml.cs

```diff
@@ -47,6 +48,11 @@ public PRMigratorBar(PackageManagerModel model)
             MigratorBar.SetResourceReference(Border.BorderBrushProperty, VsBrushes.ActiveBorderKey);
         }
 
+        public static PRMigratorBar Create(PackageManagerModel model)
```

Comment:
curious -- why the move to a factory method?

Reply:
`PackageManagerModel` now implements `IDisposable`.

I wanted to avoid conveying a transfer of ownership (and dispose responsibility) with a constructor (like prominent examples in .NET).  The lifetime of `PackageManagerModel` is managed at a higher scope, and a factory method doesn't carry the ownership implication of a constructor.

Not super successful, it seems.